internal/app/decision_maker.go

```diff
@@ -409,6 +409,9 @@ func (cs *currentState) inspectUpgradeScenario(r *release, p *plan, chartName, c
 			if diff := r.diff(); diff != "" {
 				r.upgrade(p)
 				p.addDecision("Release [ "+r.Name+" ] will be updated", r.Priority, change)
+			} else if flags.alwaysUpgrade {
```

Comment:
why not do this before the diff and avoid the unnecessary operation?

Reply:
good point.